Google.PowerShell/BigQuery/GcbqTable.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright 2015-2017 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Bigquery.v2;
+using Google.Apis.Bigquery.v2.Data;
+using Google.PowerShell.Common;
+using System;
+using System.Net;
+using System.Management.Automation;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Lists all tables in the specified dataset or returns a specific table.
+    /// </para>
+    /// <para type="description">
+    /// Lists all tables in the specified dataset. Requires the READER dataset role. If a table ID is specified, 
+    /// it will return the table resource, which describes the data in the table. Note that this is not the 
+    /// actual data from the table. If no Project is specified, the default project will be used. The dataset 
+    /// may be selected by passing in a Dataset object via pipeline or by passing the dataset ID with the 
+    /// -Dataset parameter. This cmdlet returns a TableList object if no Table was specified, and a Table otherwise.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Get-GcbqDataset “my_data” | Get-GcbqTable</code>
```

Comment:
Is it bad to have extra functionality?  It's also supported here by passing in via parameter which is the way that all of the other cmdlets will take in a dataset ID.    (And I can totally see arguments either way, just wondering what the trend is for this codebase)